python/src/tasks/gene_expression.py

```diff
@@ -31,7 +31,13 @@ def compute(self):
         )
         resultR = r.json()
         result = {}
+        if not len(resultR):
+            result[genes[0]] ={
+                "message": 404,
+            }
+            return self._format_result(result)
```

Comment:
For this part, would it be better to have it as

```
if len(resultR):
    # build result
else :
    # Return 404

return self._format_result(result)
```

So you don't need to have 2 return statements?